src/VirtoCommerce.CartModule.Core/Model/Search/ShoppingCartSearchCriteria.cs

```diff
@@ -12,5 +13,11 @@ public class ShoppingCartSearchCriteria : SearchCriteriaBase
         public string Type { get; set; }
         public string[] CustomerIds { get; set; }
         public string OrganizationId { get; set; }
+
+        public DateTime? StartDate { get; set; }
```

Comment:
Better use the following names `CreatedStartDate` and `CreatedEndDate` because the name `StartDate`  is confused which property will be used for filtration

Reply:
Fixed